packages/examples/stories/docs/figspec/README.mdx

```diff
@@ -1,13 +1,12 @@
-import { Meta } from "@storybook/addon-docs/blocks";
-import { Subtitle } from "@storybook/addon-docs/blocks";
+import { Meta, Subtitle } from "@storybook/blocks";
 import { LinkTo } from "@storybook/addon-links";
 import { Badge } from "@storybook/components";
 
 <Meta title="Docs/Figma (Figspec)/README" />
 
 # Embedding Figma designs via Figspec
 
-<Subtitle>
+<Subtitle of=".">
```

Comment:
I think this might not be correct. The Subtitle block only accepts `children` as prop

Reply:
Seems like this block was not updated to v7 standards!